scripts/manylinux/build_manylinux_wheel.sh

```diff
@@ -0,0 +1,183 @@
+      ;;
+  esac
+done
+
+
+set -o xtrace
+
+
+NUPIC_CORE_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+
+DEST_WHEELHOUSE="${NUPIC_CORE_ROOT}/nupic_bindings_wheelhouse"
+
+WHEEL_PLATFORM_NAME="manylinux1_x86_64"
+
+
+echo "RUNNING MANYLINUX BUILD: BUILD_TYPE=${BUILD_TYPE}, ROOT=${NUPIC_CORE_ROOT}"
+
+# The manylinux image provides cmake28, but doesn't symlink it to cmake
+CMAKE="cmake28"
+${CMAKE} --version
+
+# manylinux provides the required gcc toolchain for building extensions
+CC="gcc"
+CXX="g++"
+
+# Add the python 2.7 binaries from manylinux image to PATH, overriding system
+# Python
+PYBIN="/opt/python/cp27-cp27mu/bin"
+
+PATH="${PYBIN}:${PATH}"
+
+# Help python executable find its shared lib; unlike the community manylinux
+# docker image that builds statically-linked python, our custom manylinux image
+# builds python with libpython.so python library, because our current C++ tests
+# of python interfaces depend on having libpython.so.
+LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:$(dirname ${PYBIN})/lib"
+
+# Install pycapnp to get the headers capnp/helpers/checkCompiler.h, etc.
+${PYBIN}/pip install pycapnp==0.5.8
+
+# Install nupic.bindings dependencies; the nupic.core cmake build depends on
+# some of them.
+${PYBIN}/pip install -r ${NUPIC_CORE_ROOT}/bindings/py/requirements.txt
+
+#
+# Build and install nupic.bindings
+#
+
+mkdir -p ${NUPIC_CORE_ROOT}/build/scripts
+cd ${NUPIC_CORE_ROOT}/build/scripts
+
+# Configure nupic.core build
+PYTHON_INCLUDE_DIR="/opt/python/cp27-cp27mu/include/python2.7"
+
+EXTRA_CMAKE_DEFINITIONS=""
+if [[ "$BUILD_TYPE" == "Debug" ]]; then
+  EXTRA_CMAKE_DEFINITIONS="-DNUPIC_IWYU=ON -DNTA_COV_ENABLED=ON"
+fi
+
+# NOTE Pass -DPYTHON_LIBRARY to cause `find_package(PythonLibs REQUIRED)` to
+# return the desired include directory
+${CMAKE} ${NUPIC_CORE_ROOT} \
+    -DPYTHON_LIBRARY="$(dirname ${PYBIN})/lib/libpython2.7.so" \
+    -DPYTHON_INCLUDE_DIR=${PYTHON_INCLUDE_DIR} \
+    -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
+    ${EXTRA_CMAKE_DEFINITIONS} \
+    -DCMAKE_INSTALL_PREFIX=${NUPIC_CORE_ROOT}/build/release \
+    -DPY_EXTENSIONS_DIR=${NUPIC_CORE_ROOT}/bindings/py/nupic/bindings
+
+# Build nupic.core
+make install
+
+# Build the manylinux wheel
+cd ${NUPIC_CORE_ROOT}
+${PYBIN}/python setup.py bdist_wheel \
+  --dist-dir ${DEST_WHEELHOUSE} \
+  --plat-name ${WHEEL_PLATFORM_NAME}
+
+# Install the wheel that we just built
+${PYBIN}/pip install --ignore-installed ${DEST_WHEELHOUSE}/nupic.bindings-*.whl
+
+# Run the nupic.core C++ tests
+cd ${NUPIC_CORE_ROOT}/build/release/bin
+./connections_performance_test
+./cpp_region_test
+./helloregion
+./hello_sp_tp
+./prototest
+./py_region_test
+./unit_tests
+
+# Run the nupic.core python tests
+BUILD_TAG=$(echo "${BUILD_TYPE}" | tr '[:upper:]' '[:lower:]') # to lower case
+
+if [[ "$RESULT_KEY" != "" ]]; then
+  SUFFIX="$RESULT_KEY"
+else
+  SUFFIX="junit"
+fi
+
+JUNIT_OUTFILE="${NUPIC_CORE_ROOT}/bindings/py/manylinux-${BUILD_TAG}-results-${SUFFIX}.xml"
```

Comment:
Thanks for the explanation, I have to look through it again because I thought `RESULT_KEY` and `BUILD_TAG` were, at least in some cases, set to the same value (`OPTARG`)